### PR TITLE
Force protocol

### LINKED ISSUE #2: scheme-relative is absolute?
https://github.com/sindresorhus/is-absolute-url/blob/master/test.js#L9

I'd think that it should be `false`.

### PR DESCRIPTION
**Pull request by kevva**
Fixes #2.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ it('should match absolute urls', function () {
 	assert(isAbsoluteUrl('http://sindresorhus.com'));
 	assert(isAbsoluteUrl('https://sindresorhus.com'));
 	assert(isAbsoluteUrl('file://sindresorhus.com'));
-	assert(isAbsoluteUrl('//sindresorhus.com'));
```
Comment (stevemao) [inline test.js]: Instead of removing it, test the inverse?

Comment (kevva) [inline test.js]: Good point.

**Comment by stevemao** [comment]
If this is merged, we should add some notes in README because I don't think many people would expect this result.

File: readme.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ var isAbsoluteUrl = require('is-absolute-url');
 isAbsoluteUrl('http://sindresorhus.com/foo/bar');
 //=> true
 
+isAbsoluteUrl('//sindresorhus.com');
+//=> false
```
Comment (stevemao) [inline readme.md]: I think there should be a **NOTE** in the doc just to make it super clear..

Comment (arthurvr) [inline readme.md]: Not sure myself. Would an explicit note add more value than the example? Or just to emphasise it?

Comment (sindresorhus) [inline readme.md]: Yeah, I think it's should link to an explanation of why it's not an absolute path.

**Comment by sindresorhus** [comment]
ping @kevva

File: readme.md (hunk 2 above)
Comment (kevva) [inline readme.md]: Not sure if I'm the correct person to explain why it's not. I just created this PR based on the discussion in #2.

**Pull request merged**